secret_hitler/app.py

```diff
@@ -656,46 +683,73 @@ async def start_president_legislative(game : Game):
     embed.set_image(url="attachment://president.png")
     msg = await client.get_user(game.president.player_id).send(embed=embed, file=file)
 
+async def cleanup(guild):
+    # Removes everything created by the setup function
+    category = get_category(guild)
+    if category is not None:
+        for c in category.channels:
+            logger.debug("Deleting channel: " + c.name)
+            await c.delete()
+            logger.info("Deleted channel: " + c.name)
+        logger.debug("Deleting category: " + category.name)
+        await category.delete()
+        logger.info("Deletedcategory: " + category.name)
```

Comment:
nit: Typo in the log message

```suggestion
        logger.info("Deleted category: " + category.name)
```